JS/Lesson14/onder/script.js

```diff
@@ -9,71 +9,99 @@ const randomNumberFrom0To10 = Math.floor(Math.random()*10+1);
 console.log(randomNumberFrom0To10);
 // 1. Use the `length` property to find the length of a string and print it to the console.
 const stringExercise1 = 'Object-oriented programming is a powerful paradigm.';
+console.log("Exercise1: ", stringExercise1.length);
```

Comment:
Hi Önder, since this file (lesson 14) is added to this PR (lesson 15), you need to be careful with merging. 
Do it like this:
- first, merge this PR when approved
- locally, switch to the master branch and pull changes (on the main branch)
- switch to the branch that you used for the PR for lesson 14
- merge master branch into your branch 
This video should help https://youtu.be/YINamUvOYhA